packages/@ama-sdk/schematics/schematics/java/client-core/swagger-codegen-java-client/config/swagger-codegen-config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "modelPackage" : "com.amadeus.dapi",
```

Comment:
not really *agnostic* this model

Reply:
this file is already present twice in the repo. I created an issue to remove all references to DAPI/DES/Digital https://github.com/AmadeusITGroup/otter/issues/1190